### PR TITLE
feat: add apr view

### PR DESCRIPTION
**Pull request by yqz0203**
Address #47 

Add APR view in stake page.

### CODE REVIEW

File: src/services/graph/fetchers/exchange.ts [hunk 1]
```diff
@@ -158,6 +158,14 @@ export const getAvaxPrice = async (variables = undefined) => {
   // return getNativePrice(ChainId.AVALANCHE, variables)
 }
 
+export const getMistPrice = async (variables = {}) => {
+  // console.log('getSushiPrice')
```
Comment (kasumi-1) [inline src/services/graph/fetchers/exchange.ts:162]: please remove commented out debug log

File: src/services/graph/hooks/exchange.ts [hunk 2]
```diff
@@ -106,31 +107,19 @@ export function useOnePrice(variables = undefined, swrConfig: SWRConfiguration =
 
 export function useYggPrice(variables = undefined, swrConfig: SWRConfiguration = undefined) {
   const { chainId } = useActiveWeb3React()
-  const { data } = useSWR(
-    null,
-    () => getYggPrice(),
```
Comment (kasumi-1) [inline src/services/graph/hooks/exchange.ts:111]: formatting of unused functions (there are a few cases of this below) should be a separate commit

File: src/services/graph/hooks/exchange.ts [hunk 3]
```diff
@@ -188,6 +169,13 @@ export function useMaticPrice(variables = undefined, swrConfig: SWRConfiguration
 
 export function useSushiPrice(variables = undefined, swrConfig: SWRConfiguration = undefined) {
```
Comment (kasumi-1) [inline src/services/graph/hooks/exchange.ts:170]: Do we still need useSushiPrice ?

Comment (yqz0203) [reply src/services/graph/hooks/exchange.ts:170]: Now `useSushiPrice` is only referenced by `useFarmReward`, which is used in `/analytics` page. 

I can try to replace the use of `useSushiXXX` with `useMistXXX`  in `useFarmReward`.

Comment (yqz0203) [reply src/services/graph/hooks/exchange.ts:170]: I think we would better not remove `useSushiPrice`  now.  There are other code named `SuShi` in `/analytics`. The code could not be clean without removing `analytics` related pages.
